packages/react-core/src/demos/Wizard/WizardDemo.md

```diff
@@ -567,20 +372,21 @@ import {
   NavList,
   PageSection,
   PageSectionTypes,
-  PageSectionVariants
+  PageSectionVariants,
+  Wizard,
+  WizardStep
 } from '@patternfly/react-core';
-import { Wizard as WizardDeprecated } from '@patternfly/react-core/deprecated';
 import imgBrand from './imgBrand.svg';
 import BarsIcon from '@patternfly/react-icons/dist/js/icons/bars-icon';
 import DashboardWrapper from '@patternfly/react-core/src/demos/examples/DashboardWrapper';
 
-class FullPageWizard extends React.Component {
+class WizardFullPage extends React.Component {
```

Comment:
This demo has a number of unused imports as well - and an unused activeItem state variable.